interfaces/HO-Portal/src/app/services/programs-service-api.service.ts

```diff
@@ -131,11 +131,17 @@ export class ProgramsServiceApiService {
       .toPromise();
   }
 
-  getTotalIncluded(programId: number | string): Promise<TotalIncluded> {
+  getTotalTransferAmounts(
+    programId: number | string,
+    referenceIds?: string[],
+  ): Promise<TotalTransferAmounts> {
     return this.apiService
-      .get(
+      .post(
         environment.url_121_service_api,
-        `/export-metrics/total-included/${programId}`,
+        `/export-metrics/total-transfer-amounts/${programId}`,
+        {
+          referenceIds: JSON.stringify(referenceIds),
```

Comment:
Why do they need to be converted into a string explicitly?
NestJS/Angular will already do that anyway already, I think...

Related AB#12125